spec/postrank-uri_spec.rb

```diff
@@ -98,6 +98,11 @@ def n(uri)
       expect(n('http://igvita.com/a/b')).to eq('http://igvita.com/a/b')
       expect(n('http://igvita.com/a/b/')).to eq('http://igvita.com/a/b')
     end
+
+    it 'handles nested url' do
+      expect(n('http://igvita.com/a/b/http://hello.com')).to eq('http://igvita.com/a/b/http://hello.com')
+      expect(n('http://igvita.com/a/b/https://hello.com')).to eq('http://igvita.com/a/b/https://hello.com')
```

Comment:
One of the expects should have a double slash that is collapsed to a single one.